src/services/EventService.ts

```diff
@@ -2,12 +2,43 @@ import { Event, AppUser, Attendance, RSVP } from '@Entities';
 import { AttendanceService, AttendanceServiceImpl } from './AttendanceService';
 import { RSVPService, RSVPServiceImpl } from './RSVPService';
 
-import { getRepository } from 'typeorm';
-import { MultipleAttendanceQuery } from '@Payloads';
+import { getRepository, FindManyOptions, Not } from 'typeorm';
+import { MultipleAttendanceQuery, MultipleEventQuery } from '@Payloads';
 
 export class EventService {
   constructor(private attendanceService: AttendanceService, private rsvpService: RSVPService) {}
 
+  private buildMultipleEventQuery(
+    multipleEventQuery: MultipleEventQuery,
+    canShowPending: boolean
+  ): FindManyOptions<Event> {
+    const { pending, ready, complete } = multipleEventQuery;
+    const query: FindManyOptions<Event> = {};
+    const whereArr = [];
+
+    if (canShowPending) {
+      if (pending) {
+        whereArr.push({ status: 'pending' });
```

Comment:
for the 'pending' strings can we reuse the constants from where we defined them in the Event Entity file?